gonzo/clouds/compute.py

```diff
@@ -266,7 +265,16 @@ def _monkeypatch_instance(self, instance):
         instance.extra['gonzo_network_address'] = instance.extra['dns_name']
 
     def security_groups_for_launch(self, security_group_names):
-        return security_group_names
+        vpc_id = security_group_names[0].extra['vpc_id']
```

Comment:
`security_group_names` suggests a list of strings. but the can't be, because these objects have a `.extra` attribute.